includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -165,7 +142,7 @@ public function should_use_sensei_theme() {
 	public static function has_sensei_theme_enabled( int $course_id ) {
 		$theme              = get_post_meta( $course_id, self::THEME_POST_META_NAME, true );
 		$enabled_for_course = self::SENSEI_THEME === $theme;
-		$enabled_globally   = (bool) \Sensei()->settings->settings['sensei_learning_mode_all'];
+		$enabled_globally   = (bool) \Sensei()->settings->get( 'sensei_learning_mode_all' );
```

Comment:
It's a cherry-pick from `master` to fix an issue in the tests when we enabled the feature flag.